samples/net/sockets/civetweb/src/main.c

```diff
@@ -10,7 +10,14 @@
 
 #include "civetweb.h"
 
-#define CIVETWEB_MAIN_THREAD_STACK_SIZE 4096
+#define XSTR(x) STR(x)
+#define STR(x) #x
```

Comment:
Isn't there already defined macros for that in Zephyr codebase ? 

I think ```STRINGIFY()``` is the one I'm thinking of. But that might be reserved for internal code only.

Reply:
You're perfectly right. It will be changed with next push.